spec/grid_spec.rb

```diff
@@ -1,6 +1,47 @@
 require_relative "../lib/grid"
 
 RSpec.describe Grid do
+  describe "#computer_won?" do
```

Comment:
So looking at these the duplication of testing for computer won vs player won, I do want to see an independent object that can take some data about the coordinates of a certain mark, regardless of its X or O ness and figure out whether that mark won.